packages/ui/src/lib/button/CloseButton.svelte

```diff
@@ -8,11 +8,14 @@ const dispatch = createEventDispatcher<{ click: undefined }>();
 function click(): void {
   dispatch('click');
 }
+
+export let textColor = 'text-gray-800';
+export let hoverBgColor = 'bg-white';
```

Comment:
See https://github.com/podman-desktop/podman-desktop/issues/9821#issuecomment-2471192647 `Advanced HTML element genericity` maybe it will help